source/extensions/filters/network/common/redis/supported_commands.h

```diff
@@ -64,6 +65,20 @@ struct SupportedCommands {
    * @return ping command
    */
   static const std::string& ping() { CONSTRUCT_ON_FIRST_USE(std::string, "ping"); }
+
+  /**
+   * @return commands which alters the state of redis
+   */
+  static const std::set<std::string>& writeCommands() {
```

Comment:
I would use `absl::flat_hash_set` here. It will be faster. Feel free to replace other similar examples in the redis code.

Reply:
done